internal/config/validation.go

```diff
@@ -55,22 +55,35 @@ func translateInvalidK8sVersionError(ut ut.Translator, fe validator.FieldError)
 	validVersionsSorted := semver.ByVersion(validVersions)
 	sort.Sort(validVersionsSorted)
 
-	errorMsg := fmt.Sprintf("Supported versions: %s", strings.Join(validVersionsSorted, " "))
+	maxVersion := validVersionsSorted[len(validVersionsSorted)-1]
+	minVersion := validVersionsSorted[0]
```

Comment:
Technically, this can panic if `len(validVersionsSorted) == 0`. On the other hand, it's hard-coded anyway.
So not sure whether to do a length check here or not. You decide :)

Reply:
Dunno. Added a check :D